management/user.go

```diff
@@ -65,6 +65,9 @@ type User struct {
 	// AppMetadata holds data that the user has read-only access to (e.g. roles,
 	// permissions, vip, etc).
 	AppMetadata map[string]interface{} `json:"app_metadata,omitempty"`
+
+	// The user's picture url
+	Picture *string `json:"picture,omitempty"`
```

Comment:
Could you amend the tests to verify this functionality?

Reply:
@alexkappa Done.